src/main/java/org/broadinstitute/consent/http/enumeration/VoteType.java

```diff
@@ -1,8 +1,10 @@
 package org.broadinstitute.consent.http.enumeration;
 
+import org.broadinstitute.consent.http.resources.Resource;
+
 public enum VoteType {
 
-    DAC("DAC"), FINAL("FINAL"), AGREEMENT("AGREEMENT"), CHAIRPERSON("CHAIRPERSON"), DATA_OWNER("DATA_OWNER");
+    DAC("DAC"), FINAL("FINAL"), AGREEMENT("AGREEMENT"), CHAIRPERSON(Resource.CHAIRPERSON), DATA_OWNER("DATA_OWNER");
```

Comment:
Note that the vote type's `DATA_OWNER` is not consistent with the role we define in the roles table.